robolectric/src/test/java/org/robolectric/RobolectricTestRunnerTest.java

```diff
@@ -125,8 +124,8 @@ public void shouldReportPerfStats() throws Exception {
 
     assertThat(collectors).hasSize(2);
     PerfStatsCollector perfStatsCollector = collectors.get(0);
-    assertThat(perfStatsCollector.getDescription())
-        .isEqualTo("first(" + TestWithTwoMethods.class.getName() + ")");
+    // assertThat(perfStatsCollector.getTestMethod())
```

Comment:
What i purpose of keeping commented out code when there's huge git commits history advantage? :-) If I were you, I would remove it instead of leaving as a comment.

Reply:
Yep, oversight, thanks!